track/targets.py

```diff
@@ -5,6 +5,7 @@
 from abc import abstractmethod, ABC
 from functools import lru_cache
 from datetime import datetime
+import time
```

Comment:
- [ ] Changes in this file look good, but should probably be in a separate commit since this feature can stand alone and has utility outside of guidescope alignment automation.